goals.md

```diff
@@ -9,29 +9,29 @@ Based on this, we would like to bring the goals to the TODO Group for review and
 
 The goals will be revisited and updated yearly.
 
-## 2022
+## [PROPOSED] 2022
 
-### 1. Cultivate growth and adoption of OSPOs
+### 1 Cultivate growth and adoption of OSPOs
 
-* Release at least 6 new OSPO case studies
-* Launch 6 new OSPO guides
-* Host 12 OSPOlogy sessions
-* Build out and run OSPO Survey 2022
-* Create and publish a OSPO Maturity Model
+- Release at least 6 new OSPO case studies
+- Launch 6 new OSPO guides
+- Host 12 OSPOlogy sessions
+- Build out and run OSPO Survey 2022
+- Create and publish a OSPO Maturity Model
 
-## 2. Grow the membership base
+## 2 Grow the membership base
 
-* Grow TODO membership by 20%
-* Cultivate relationships with other OSPO adjacent communities like CHAOSS
-* Grow OSPO Ambassadors by 
+- Grow TODO membership by 20%
+- Cultivate relationships with other OSPO adjacent communities like CHAOSS
+- Grow OSPO Ambassadors by 
 
-## 3. Create value for all members
+## 3 Create value for all members
 
-* Run at least 2 surveys to get feedback, >80% satisfaction
-* Improve and refresh TODO Website, increase traffic by 20%
-* Host at least 2 OSPOCons
-* Host first OSPOCon in APAC
-* Break down silos and enhance communication across channels (slack, mailing lists, Social media)
+- Run at least 2 surveys to get feedback, >80% satisfaction
+- Improve and refresh TODO Website, increase traffic by 20%
```

Comment:
Before we commit to this, I'd like more information about what constitutes a 20% increase. Is this realistic? A stretch goal? (which is to say: what's the current traffic?)

Reply:
The goal "Improve and refresh TODO Website" comes from complaints from some community folks about the current website UX and UI. The website is old, resources are hard to find, some resources are outdated, contribution guidelines are not clear, it doesn't show the existing TODO Chapters such as the EU Chapter, and is not SEO optimized.

I think the 20% indicator is just a way to make a goal measurable and more specific. IMHO and overall speaking, if the website is updated and improves its UI and UX, we can ease access to OSPO resources and improve shareability to the broader community, breaking down silos between the TODO General Members and OSPO Community.

Reply:
Based on @geekygirldawn's last comment is that this is more a topic of finding a way to measure visitor's engagement and not traffic. If that's the case, I see one possible approach:

Analyze specific page bounce rates (e.g number of people leaving todogroup.org/guides after visiting the page). If we get the bounce rate, we can easily get its inverse, the "engagement rate" (people that actually engage in the website)

Cons: TODO Website currently runs on netlify and uses Netlify Analytics (which does not show this metric, AFAIK).